protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -35,12 +35,24 @@ func BuildEtcdManifest(c *EtcdCluster) *v1.Pod {
 		"k8s-app": c.PodName,
 	}
 
+	// TODO another hardcoded version
+	image := "/etcd:2.2.1"
+	imageRegistry := "gcr.io/google_containers"
+
+	// Test to determine if the container registry has been passed in as a flag.
+	// If so use the provider registry location.
+	if c.ImageSource == "" {
```

Comment:
We could but we are moving where the version is hard coded at as well then.  I have an issue with all the hard coded version stuff we have running around.  Ideas?